src/notation/imasternotation.h

```diff
@@ -41,7 +41,8 @@ class IMasterNotation
     virtual Ret createNew(const ScoreCreateOptions& scoreInfo) = 0;
     virtual RetVal<bool> created() const = 0;
 
-    virtual Ret save(const io::path& path = io::path()) = 0;
+    virtual Ret save(const io::path& path = io::path(), SaveMode saveMode = SaveMode::Unknown) = 0;
+    virtual Ret saveSelection(const io::path& path = io::path()) = 0;
```

Comment:
I think, better to implement this using one method. We can create a special enum to separate two different modes

Reply:
thanks, fixed